src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -412,6 +412,7 @@
     <OutOfBoxFramework Include="netcore50">
       <RuntimeId>win10</RuntimeId>
     </OutOfBoxFramework>
+    <OutOfBoxFramework Condition="'$(IsUap101Defined)'=='true'" Include="uap10.1"/>
```

Comment:
Is it not better to add the OutOfBoxFramework item in corefx as the temporary measure instead of introducing this property?